### PR TITLE
Update erc-20 developer docs (French)

### PR DESCRIPTION
**Pull request by AsheBarrett**
## Description

Added caps to the second list of item for more consistency.
Slightly 
Updated the example definition to be more friendly instead of being directly translated by a machine to ease the comprehension.

### CODE REVIEW

File: src/content/translations/fr/developers/docs/standards/tokens/erc-20/index.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ event Approval(address indexed _owner, address indexed _spender, uint256 _value)
 
 ### Exemples {#web3py-example}
 
-Voyons comment une norme est si importante pour nous faciliter le contrôle de tout contrat de jeton ERC-20 sur Ethereum. Nous avons juste besoin de l'interface binaire-programme (ABI) du contrat pour créer une interface à n'importe quel jeton ERC-20. Comme vous pouvez le voir ci-dessous, nous utiliserons une ABI simplifiée, pour en faire un exemple de faible friction.
+Voyons pourquoi une norme est importante et pourquoi elle nous facilitee le contrôle de tout contrat de jeton ERC-20 sur Ethereum. Nous avons juste besoin de l'interface binaire-programme (ABI) du contrat pour créer une interface à n'importe quel jeton ERC-20. Comme vous pouvez le voir ci-dessous, nous utiliserons une ABI simplifiée, pour en faire un exemple facile à comprendre.
```
Comment (vdusart) [inline src/content/translations/fr/developers/docs/standards/tokens/erc-20/index.md:71]:
```suggestion
Voyons pourquoi une norme est importante et pourquoi elle nous facilite le contrôle de tout contrat de jeton ERC-20 sur Ethereum. Nous avons juste besoin de l'interface binaire-programme (ABI) du contrat pour créer une interface à n'importe quel jeton ERC-20. Comme vous pouvez le voir ci-dessous, nous utiliserons une ABI simplifiée, pour en faire un exemple facile à comprendre.
```

Comment (MATsxm) [reply src/content/translations/fr/developers/docs/standards/tokens/erc-20/index.md:71]: +1 with @vdusart but stay on my POV (see FR Discord chan)

**Review by minimalsm — APPROVED** [review]
Thanks again folks @AsheBarrett @MATsxm @vdusart

**Comment by AsheBarrett** [comment]
Hello there!

Sadly it doesn't seem there is anything to mint here 😕

Regards,

Ashe
________________________________
De : gitpoap-bot[bot] ***@***.***>
Envoyé : lundi 7 novembre 2022 13:18
À : ethereum/ethereum-org-website ***@***.***>
Cc : AsheBarrett ***@***.***>; Mention ***@***.***>
Objet : Re: [ethereum/ethereum-org-website] Update erc-20 developer docs (French) (PR #8428)


Congrats, your important contribution to this open-source project has earned you a GitPOAP!

GitPOAP: 2022 Ethereum.org Contributor<https://www.gitpoap.io/gp/74>:

[GitPOAP: 2022 Ethereum.org Contributor GitPOAP Badge]<https://camo.githubusercontent.com/8911b837821b25e120ecee46d7c573eddd863acf8f491efd0029cc20ad74ac4c/68747470733a2f2f6173736574732e706f61702e78797a2f676974706f61702d323032322d657468657265756d6f72672d636f6e7472696275746f722d323032322d6c6f676f2d313634393237303230333737302e706e67>

Head to gitpoap.io<https://www.gitpoap.io> & connect your GitHub account to mint!

Learn more about GitPOAPs here<https://medium.com/gitpoap/whats-a-gitpoap-72f1f5778c6/>.

**Pull request merged**